rumydata/_base.py

```diff
@@ -229,7 +229,8 @@ def _flatten_exceptions(cls, error):
         checking to see if any of those errors contain additional errors, then
         continuing to recurse until all errors have been yielded.
         """
-        yield error
-        for el in error._errors:
-            for x in cls._flatten_exceptions(el):
-                yield x
+        if error:
```

Comment:
The problem I was running into was a specific test I created ended up failing because the way it was being called wasn't producing an error... so it was trying to reference `_errors` from `None`, and failing. I suspect the right way to get around the problem would be to modify the test to generate an error appropriately